example/spec/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -43,11 +43,11 @@ describe('The S3 Ingest Granules workflow', () => {
       lambdaOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'SyncGranule');
     });
 
-    it('has expected payload', () => {
+    it('outputs 1 granule and 2 files in the payload', () => {
       expect(lambdaOutput.payload).toEqual(expectedSyncGranulePayload);
```

Comment:
I'm being a little nitpicky here, but this isn't quite the right level of detail here.  The `describe` and `it` statement should correspond roughly to acceptance criteria on a ticket.  So this would be something like `it("includes the ingested granules and its matching files in its output")` but the assertions would be specific to the particular test data.